meilisearch-core/Cargo.toml

```diff
@@ -20,19 +21,23 @@ heed = "0.6.1"
 indexmap = { version = "1.2.0", features = ["serde-1"] }
 intervaltree = "0.2.5"
 itertools = "0.8.2"
+lazy_static = "1.4.0"
 levenshtein_automata = { version = "0.1.1", features = ["fst_automaton"] }
 log = "0.4.8"
 meilisearch-schema = { path = "../meilisearch-schema", version = "0.9.0" }
 meilisearch-tokenizer = { path = "../meilisearch-tokenizer", version = "0.9.0" }
 meilisearch-types = { path = "../meilisearch-types", version = "0.9.0" }
 once_cell = "1.2.0"
 ordered-float = { version = "1.0.2", features = ["serde"] }
+pest = {git = "https://github.com/MarinPostma/pest.git"} 
+pest_derive = "2.0"
```

Comment:
Can't we move this dependency to the `dev-dependencies` section?

Reply:
It appears so, rust complains if I put it as a devdep